application.py

```diff
@@ -148,6 +148,8 @@ def callback():
         current_user.id = int(unique_id)
     else:
         return "User email not available or not verified by Google", 400
+    # WARNING: HACK THIS METHOD
+    unique_id = unique_id[3:]
```

Comment:
checkout this post
https://dba.stackexchange.com/questions/51536/datatype-bigger-than-bigint

I think if we change it to decimal we can host more digits.

Reply:
@linxiaow I have changed for you. Please see commit below.